channeld/channeld.c

```diff
@@ -1564,6 +1587,8 @@ static void handle_peer_commit_sig(struct peer *peer, const u8 *msg)
 	const struct htlc **htlc_map, **changed_htlcs;
 	const u8 *funding_wscript;
 	size_t i;
+	struct simple_htlc **htlcs;
+	const u8 * msg2;
```

Comment:
maybe just `msg`?

Reply:
I notice that in `handle_peer_revoke_and_ack`, the existing code uses a descriptive name `revocation_msg` to disambiguate.  Probably best to follow that convention.